pages/emotion/[id]/edit.js

```diff
@@ -0,0 +1,26 @@
+import EmotionForm from "@/components/EmotionForm";
+import Heading from "@/components/Heading";
+import { useRouter } from "next/router";
+
+export default function EditPage({ objects, onUpdateEmotion }) {
+  const router = useRouter();
+  const { id } = router.query;
+
+  const existingEmotion = objects.find((object) => object.id === id);
+  if (!existingEmotion) {
+    return <p>Loading...</p>;
+  }
+
+  return (
+    <>
+      <Heading>Edit Emotion</Heading>
```

Comment:
I don't think you need this heading, since you have already defined a heading in the form ("Update your Emotion:")

Reply:
Thanks for your advice. We will discuss the removal in our team and might get back to this on one of our next user stories